yt_dlp/extractor/rutube.py

```diff
@@ -90,7 +93,7 @@ def _download_and_extract_formats(self, video_id, query=None):
 class RutubeIE(RutubeBaseIE):
     IE_NAME = 'rutube'
     IE_DESC = 'Rutube videos'
-    _VALID_URL = r'https?://rutube\.ru/(?:video(?:/private)?|(?:play/)?embed)/(?P<id>[\da-z]{32})'
+    _VALID_URL = r'https?://rutube\.ru/(?:video(?:/private)?|(?:play/)?embed|live/video)/(?P<id>[\da-z]{32})'
```

Comment:
As `https://rutube.ru/live/video/private/c58f502c7bb34a8fcdd976b221fca292/` is also valid, I would do:
```suggestion
    _VALID_URL = r'https?://rutube\.ru/(?:(?:live/)?video(?:/private)?|(?:play/)?embed)/(?P<id>[\da-z]{32})'
```